tectonic/core-bridge.c

```diff
@@ -69,7 +71,7 @@ dvipdfmx_simple_main(tt_bridge_api_t *api, char *dviname, char *pdfname)
 {
     extern int dvipdfmx_main(int argc, char *argv[]);
 
-    char *argv[] = { "dvipdfmx", "-o", pdfname, dviname };
+    char *argv[] = { xstrdup("dvipdfmx"), xstrdup("-o"), pdfname, dviname };
```

Comment:
OK, yes, that was my hunch. It would be much better to const-ify the variables rather than perform extra mallocs, if that's possible — which I think it shoud be in the cases where it came up here.